test/test_requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest
+geopandas
+matplotlib
```

Comment:
I'd prefer the tests not run on this all the time since some users don't want heavy dependencies like geopandas or matplotlib installed on their machines

Reply:
mmm those are optional and separated from the main `requirements.txt` because of that and you just install them if you are running the tests.

In general I prefer that everything needed to develop is in a requirements file.

Reply:
I agree, but those aren't requirements for using the package, just some of the features of the package. Travis takes care of setting up an environment that has them conditionally: https://github.com/CartoDB/cartoframes/blob/master/.travis.yml#L13-L14

Reply:
Oh I see, so you mean, those specific tests that require those dependencies are (were) only ran in Travis?

Reply:
Yeah, each travis job conditionally loaded them depending on the setting in the `env` param

Reply:
Ohh, now I get it. Sorry for the confusion then 🙇 

I'm adding a new issue to revert this in the future, for now (while we are developing) I guess it's fine as it is now.